server/plugin.go

```diff
@@ -41,10 +43,13 @@ const (
 	SettingOff           = "off"
 
 	chimeraGitLabAppIdentifier = "plugin-gitlab"
+
+	invalidTokenError = "401 {error: invalid_token}" //#nosec G101 -- False positive
 )
 
 var (
-	manifest model.Manifest = root.Manifest
+	manifest   model.Manifest = root.Manifest
+	tokenMutex sync.Mutex
```

Comment:
Note that we'll want to use `cluster.Mutex` instead of this mutex

Reply:
I figured the regular mutex would perform better (maybe ? because the cluster one uses KV) and possibly be more stable. Do we really need the mutex to be user specific if it's only happening during the refresh? Or do we still need the cluster one to support cluster instances?

Reply:
Committed the change to use `cluster.Mutex`